config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -0,0 +1,92 @@
+presubmits:
+  containerd/containerd:
+  - name: pull-containerd-build
+    always_run: false
+    optional: true
+    branches:
+    - master
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-containerd-io
+      testgrid-tab-name: pull-containerd-build
+      description: build artifacts
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200901-eeeadc5-master
+        command:
+        - runner.sh
+        args:
+        - ./test/build.sh
```

Comment:
will need to add a script in containerd/containerd or move to make..

Reply:
Correct. we can tweak this later.